py/vtdb/grpc_vtgate_client.py

```diff
@@ -118,7 +116,8 @@ def rollback(self):
     finally:
       self.session = None
 
-  @vtgate_utils.exponential_backoff_retry((dbexceptions.TransientError))
+  @vtgate_utils.exponential_backoff_retry((dbexceptions.ThrottledError,
```

Comment:
Yes it is. Just making this consistent though, that's what we do in other clients too.